### ISSUE TITLE
Add a new sql - fte_total_current

### ISSUE THREAD

**Issue by andreabr-dixie** [dsu-effectiveness/utDataStoR#26]
This new sql will match what is coming out of banner report (fte_off_rept.sql).

`WITH CTE_graduate_fte AS
    (SELECT a.term_id,
            ROUND(SUM(a.attempted_credits)/10, 2) as grad_attmp_cred_fte
       FROM export.student_section_version a
      WHERE a.is_enrolled = TRUE
        AND a.version_desc = 'Current'
        AND a.course_level_id = 'GR'
     GROUP BY a.term_id
     ),

CTE_undergrad_fte AS
    (SELECT a.term_id,
            ROUND(SUM(a.attempted_credits)/15, 2) as ugrad_attmp_cred_fte
       FROM export.student_section_version a
      WHERE a.is_enrolled = TRUE
        AND a.version_desc = 'Current'
        AND a.course_level_id = 'UG'
     GROUP BY a.term_id
    )

SELECT a.term_id,
       b.term_desc,
       b.season,
       d.ugrad_attmp_cred_fte,
       coalesce(c.grad_attmp_cred_fte, 0) AS grad_attmp_cred_fte,
       ROUND(coalesce(c.grad_attmp_cred_fte, 0) + d.ugrad_attmp_cred_fte,2) as attmp_cred_fte,
       SUM(a.attempted_credits) as attmp_cred,
       a.version_desc
     FROM export.student_section_version a
LEFT JOIN export.term b
       ON a.term_id = b.term_id
LEFT JOIN CTE_graduate_fte c
       ON a.term_id = c.term_id
LEFT JOIN CTE_undergrad_fte d
       ON a.term_id = d.term_id
    WHERE a.is_enrolled = TRUE
      AND a.version_desc = 'Current'
      AND DATE_PART('year', NOW()) - b.academic_year_code :: INT <= 5
GROUP BY a.term_id, b.season, b.term_desc, c.grad_attmp_cred_fte, d.ugrad_attmp_cred_fte, a.version_desc;`

**Comment by MattNickodemus** [comment]
What is this for? Is this for a metric? Is it for a report? Please provide more detail when you are submitting issues.

**Comment by andreabr-dixie** [comment]
Good point. I don't have a good answer. As I was reviewing the FTE sql, I noticed that we had an sql that pulled off of rept (fte_off_rept.sql). As I was trying to make sure that the data was correct, I wanted to pull the same data from Edify (that is how I found out that fte_off_edify.sql was wrong), so, this query is the matching query to fte_off_rept.sql. This would replace fte_off_edify.sql, but it's name would look more like the other sql's coming from edify (to match fte_total_census).
